string color # color that has been pressed
uint32 midi_key # Key of the color that has been pressed
float32 delta_time # time since the last note was pressed